src/Uno.UWP/Graphics/Display/DisplayInformation.wasm.cs

```diff
@@ -0,0 +1,55 @@
+﻿#if __WASM__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.Foundation;
+
+namespace Windows.Graphics.Display
+{
+	public sealed partial class DisplayInformation
+	{
+
+		partial void Initialize()
+		{
+			UpdateProperties();
+		}
+
+		private void UpdateProperties()
+		{
+			UpdateLogicalProperties();
+			UpdateRawProperties();
+		}
+
+		private void UpdateLogicalProperties()
+		{
+			if (ReadJSFloat("window.devicePixelRatio", out float devicePixelRatio))
+			{
+				LogicalDpi = devicePixelRatio * 96.0f;
```

Comment:
Why `96`? Green needed. Should be extracted to a constant if it's 'the same 96' below.